CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ command line for details.
 
 ## [Unreleased]
 
+## [0.6] - 2016-01-02
+
+- Add `DockerSpawner.format_volume_name` for custom strategies for mounted volume names.
```

Comment:
@minrk I wasn't sure as this reads which 'custom strategies' that this refers to as written. Perhaps:

`for custom naming strategies for mounted volumes.`